ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/Defaults.scala

```diff
@@ -11,4 +11,8 @@ object Defaults {
 
   val TimeoutScaleFactor: Double = 1.0
 
+  // Neither ledgers nor participants scale perfectly with the number of processors.
+  // We therefore limit the maximum number of concurrent tests, to avoid overwhelming the ledger.
+  val ConcurrentRuns: Int = List(Runtime.getRuntime.availableProcessors(), 4).min
```

Comment:
You can use `sys.runtime`, and `Int#min`.

```suggestion
  val ConcurrentRuns: Int = sys.runtime.availableProcessors min 4
```